fdbserver/workloads/PhysicalShardMove.actor.cpp

```diff
@@ -267,49 +266,28 @@ struct PhysicalShardMoveWorkLoad : TestWorkload {
 		state std::string checkpointDir = abspath("fetchedCheckpoints");
```

Comment:
Can this function be called currently? If so, you might need to randomize the path.

Reply:
It won't be called concurrently, but I added a random suffix anyway.